tasks/deploy/gnosis-delegate.ts

```diff
@@ -0,0 +1,38 @@
+import { assert } from "chai";
+import { ethers } from "ethers";
+import { task, types } from "hardhat/config";
+import { HardhatRuntimeEnvironment, TaskArguments } from "hardhat/types";
+import { Account, ERC20, MockERC20__factory, GnosisBridgeSender, GnosisBridgeSender__factory } from "../../types";
+import { getSigner } from "../utils";
+import { chainIds } from "../../hardhat.config";
+import { ZERO_ADDRESS } from "../../test-utils/constants";
+import { deployGnosisBridgeSender } from "../../scripts/deployBridgeDelegates";
+import { waitForTx } from "../../tasks/utils";
+import { logContracts } from "../utils/deploy-utils";
+
+const debug = true;
+let waitForBlocks = 3;
+
+task("deploy:gnosis:sender").setAction(async function (tskArgs: TaskArguments, hre) {
+    const deployer = await getSigner(hre);
+    const waitForBlocks = tskArgs.wait;
+    const balOnGno: string = "0x7eF541E2a22058048904fE5744f9c7E4C57AF717";
+    let delegate = await deployGnosisBridgeSender(hre, deployer, balOnGno);
+    console.log(delegate.address);
+});
+
+task("deploy:gnosis:setL1ReceiverAndForwardBal").setAction(async function (tskArgs: TaskArguments, hre) {
+    const deployer = await getSigner(hre);
+    const waitForBlocks = tskArgs.wait;
+    const balOnGno: string = "0x7eF541E2a22058048904fE5744f9c7E4C57AF717";
+    const dao: string = "0x5feA4413E3Cc5Cf3A29a49dB41ac0c24850417a0";
+    const delegate: string = "0x2F70BF8d130aace466abBcbd21d34BB1A6a12c5d";
+    let crv = MockERC20__factory.connect(balOnGno, deployer);
```

Comment:
Might get a linting error here because `crv` is never reassigned? And can probably stick those config addresses in a config file, gnosis-config.ts that is a `SidechainConfig` type with these as option types on `SidechainAddresses`.